dbms/src/Storages/Page/V3/spacemap/SpaceMapSTDMap.h

```diff
@@ -55,6 +55,8 @@ class STDMapSpaceMap
         : SpaceMap(start, end, SMAP64_STD_MAP)
     {
         free_map.insert({start, end});
+        std::set<UInt64> offsets{start};
+        free_map_invert_index.emplace(end, offsets);
```

Comment:
`end` is not exactly equal to "size"? Maybe `end - start` is more accurate?

Reply:
Fixed.